# Description: Disable tooltip for login/password input line. It is confusing if it pops up automatically (depending on screen resolution this happens if the input box is in the screen center), and not all that helpful.
# Ubuntu: https://launchpad.net/bugs/331448
Index: gdm-2.20.8/gui/greeter/greeter_canvas_item.c
===================================================================
--- gdm-2.20.8.orig/gui/greeter/greeter_canvas_item.c	2009-02-24 14:04:45.000000000 +0100
+++ gdm-2.20.8/gui/greeter/greeter_canvas_item.c	2009-02-24 14:07:38.000000000 +0100
@@ -281,7 +281,6 @@
   int i;
   GtkAllocation rect;
   char *text;
-  GtkTooltips *tooltips;
   char *num_locale;
   GdkColor c;
 
@@ -493,15 +492,6 @@
 	    greeter_item_register_action_callback ("options_button",
 						   (ActionFunc)greeter_options_handler,
 						   item->data.text.menubar);
-	    
-	    /* Here add a tooltip, so that the user knows about F10 */
-	    tooltips = gtk_tooltips_new ();
-	    gtk_tooltips_set_tip (tooltips, GTK_WIDGET (entry),
-				  _("Answer questions here and press Enter "
-				    "when done.  For a menu press F10."),
-				  NULL);
-
-	    /* FIXME: how to make this accessible??? */
     }
 
     item->item = gnome_canvas_item_new (group,
